app/src/main/res/layout/fragment_text_hero.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we include this to the same fragmet we use for texts?

Reply:
We currently already have various fragments for text. This follows the split for compose where it'll be hero, heading & body. I should add it as a screenshot for the docs, though!